networks/http/http.go

```diff
@@ -129,6 +132,16 @@ func (n *Network) RoundNumber(t time.Time) uint64 {
 	return n.client.RoundAt(t)
 }
 
+// SwitchChainHash allows to start using another chainhash on the same host network
+func (n *Network) SwitchChainHash(new string) error {
+	test, err := NewNetwork(n.host, new)
```

Comment:
`test` is a funny name

Reply:
it's about testing that the new network works. Any better suggestion?

Reply:
would have suggested `network`, but `test` is fine, just a nit